src/views/List.jsx

```diff
@@ -23,9 +25,23 @@ export function List({ data }) {
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			{data.length ? (
+				<p>
+					Add additional items on the
+					<button
+						aria-label="Navigate to Manage List page"
+						onClick={() => navigate('/manage-list')}
+						className="code-button"
+					>
+						/Manage List
```

Comment:
Nitpick: You might consider removing the / from this button. Typically, users care less about the URL path of the page they're going to and more about the name.